WebAPI/Models/ServicesDTO.cs

```diff
@@ -0,0 +1,17 @@
+using EntityModel.Helpers;
+using Newtonsoft.Json;
+
+namespace WebAPI.Models
+{
+    public class ServicesDTO
+    {
+        [JsonProperty(PropertyName = "id")]
+        public string Id { get; set; }
+
+        [JsonProperty(PropertyName = "name")]
+        public string Name { get; set; }
+
+        [JsonProperty(PropertyName = "tira_servicetype")]
```

Comment:
For these models I don't think you need to worry about the json property names. We need that on the other ones for when we connect to dynamics, but these models won't interact with dynamics